cmd/osv-scanner/main_test.go

```diff
@@ -113,15 +115,29 @@ func TestRun(t *testing.T) {
         No package sources found, --help for usage information.
 			`,
 		},
+
 		{
 			name:         "",
-			args:         []string{"", "--version"},
```

Comment:
Why is this test being removed?

Reply:
Oops, replaced the wrong test, undid this change now.

Reply:
Added more tests